### ISSUE TITLE
Sentient disease adaption menu is not opening

### ISSUE THREAD

**Issue by MegaEmpirical** [yogstation13/Yogstation#7706]
Round: 27417

Both me and another player were having this issue.

**Comment by boodaliboo** [comment]
weird it worked for me last week shouldn't of been any prs effecting it

**Comment by MegaEmpirical** [comment]
Yeah it was fine for me not long ago too. I think it's an inconsistent bug.

**Comment by MegaEmpirical** [comment]
Still happening (round 34052), affected me. Closed out and still nothing.

**Comment by redmoogle** [comment]
Alright, so it seems like theres a null mob being somehow added to the disease, causing a few runtimes, and probably breaking anything it touches
